exercises/atbash-cipher/.meta/solutions/atbash_cipher.rb

```diff
@@ -1,16 +1,24 @@
-class Atbash
+class AtbashCipher
   def self.encode(plaintext)
     new(plaintext).encode
   end
 
-  attr_reader :plaintext
+  def self.decode(ciphertext)
+    new(ciphertext).decode
+  end
+
+  attr_reader :text
```

Comment:
Given that `decode` is certainly not passed un-enciphered text I thought it would be better to reduce this instance variable's name down to the common base of `plaintext` and `ciphertext`.